ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -440,13 +445,27 @@ export default defineComponent({
     }
 
     const parseSubmittingPartyInfo = (): void => {
-      const submittingParty = {
-        businessName: localState.accountInfo.name,
-        address: localState.accountInfo.mailingAddress,
-        emailAddress: localState.accountInfo.accountAdmin.email,
-        phoneNumber: localState.accountInfo.accountAdmin.phone,
-        phoneExtension: localState.accountInfo.accountAdmin.phoneExtension
-      } as SubmittingPartyIF
+      var submittingParty = {} as SubmittingPartyIF
+      if (isRoleQualifiedSupplier.value) {
+        submittingParty.businessName = localState.accountInfo?.name
+        submittingParty.address = localState.accountInfo?.mailingAddress
+        submittingParty.emailAddress = localState.accountInfo?.accountAdmin?.email
+        submittingParty.phoneNumber = localState.accountInfo?.accountAdmin?.phone
+        submittingParty.phoneExtension = localState.accountInfo?.accountAdmin?.phoneExtension
+      } else {
+        submittingParty = getMhrTransferSubmittingParty.value
+        const localAccountInfo = {} as AccountInfoIF
+        localAccountInfo.name = getMhrTransferSubmittingParty.value.businessName
+        localAccountInfo.mailingAddress = getMhrTransferSubmittingParty.value.address
+        localAccountInfo.accountAdmin = {
+          firstName: getMhrTransferSubmittingParty.value.personName.firstName,
+          lastName: getMhrTransferSubmittingParty.value.personName.LastName,
+          email: getMhrTransferSubmittingParty.value.emailAddress,
+          phone: getMhrTransferSubmittingParty.value.phoneNumber,
+          phoneExtension: getMhrTransferSubmittingParty.value.phoneExtension
+        }
+        localState.accountInfo = localAccountInfo
```

Comment:
Bit confused whats happening here.
Why are we mapping the submitting party data to `accountInfo`?

Also, i don't believe you need a const, should just be able to map directly to `localState.accountInfo`
ie `localState.accountInfo = { <mapping here> }`

Reply:
I will enter a comment in code to explain.  

I only added this code because we are unable to save a transfer without a  Submitting Party... 

The  AcccountInfo component is being used to display submitting party on the review section.  I believe this was done because we were using the auth api  to get account information to determine if user is a Qualified supplier.  

The code above ensures that the account info component is populated so it is displayed...

Reply:
There is supposed to be another ticket to allow for editing submitting party, which I believe will require this code to be refactored.

Reply:
I made the change to localState.accountInfo as suggested.